src/components/Arjuntool/Arjuntool.tsx

```diff
@@ -14,12 +14,14 @@ const description_userguide =
     "It is a multi-threaded application, can handle rate limits and supports GET,POST,XML and JSON methods. " +
     " \n\nKali's Arjun Information Page: https://www.kali.org/tools/arjun/ \n\nHow to use Arjun:\n\nStep 1: Enter a valid URL.\n" +
     "       E.g. https://www.deakin.edu.au\n\nStep 2: Enter an Optional Json Output filename.\n        E.g. arjunoutput " +
-    "\n\nStep 3: Click the scan option to commence scan. " +
-    "\n\nStep 4: View the Output block below to view the results of the tool's execution.";
+    "\n\nStep 3: Turn on the stability function if you want Arjun to prioritise stability over speed (can take a long time, so get a coffee if using this functionality)." +
+    "\n\nStep 4: Click the scan option to commence scan. " +
+    "\n\nStep 5: View the Output block below to view the results of the tool's execution.";
```

Comment:
- [ ] Remove unnecessary capitalization.

Reply:
Have done this. I also went through the tooltip again and made some slight changes to improve readability.